Mage.Sets/src/mage/cards/c/Celebr8000.java

```diff
@@ -0,0 +1,117 @@
+import mage.abilities.Ability;
+import mage.abilities.common.BeginningOfCombatTriggeredAbility;
+import mage.abilities.effects.ContinuousEffect;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.continuous.BoostSourceEffect;
+import mage.abilities.effects.common.continuous.GainAbilitySourceEffect;
+import mage.abilities.keyword.*;
+import mage.constants.*;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.game.Game;
+import mage.players.Player;
+
+/**
+ *
+ * @author jimga150
+ */
+public final class Celebr8000 extends CardImpl {
+
+    public Celebr8000(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ARTIFACT, CardType.CREATURE}, "{5}");
+        
+        this.subtype.add(SubType.CLOWN);
+        this.subtype.add(SubType.ROBOT);
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(3);
+
+        // At the beginning of combat on your turn, roll two six-sided dice.
+        // For each result of 1, Celebr-8000 gets +1/+1 until end of turn.
+        // For each other result, it gains the indicated ability until end of turn.
+        // If you rolled doubles, it also gains double strike until end of turn.
+        // * 2 -- menace
+        // * 3 -- vigilance
+        // * 4 -- lifelink
+        // * 5 -- flying
+        // * 6 -- indestructible
+        this.addAbility(
+                new BeginningOfCombatTriggeredAbility(new Celebr8000Effect(), TargetController.YOU, false));
+    }
+
+    private Celebr8000(final Celebr8000 card) {
+        super(card);
+    }
+
+    @Override
+    public Celebr8000 copy() {
+        return new Celebr8000(this);
+    }
+}
+
+class Celebr8000Effect extends OneShotEffect {
+
+    private static final ContinuousEffect[] d6_effects = {
+            /* 0 */null,
+            /* 1 */new BoostSourceEffect(1, 1, Duration.EndOfTurn),
+            /* 2 */new GainAbilitySourceEffect(new MenaceAbility(), Duration.EndOfTurn),
+            /* 3 */new GainAbilitySourceEffect(VigilanceAbility.getInstance(), Duration.EndOfTurn),
+            /* 4 */new GainAbilitySourceEffect(LifelinkAbility.getInstance(), Duration.EndOfTurn),
+            /* 5 */new GainAbilitySourceEffect(FlyingAbility.getInstance(), Duration.EndOfTurn),
+            /* 6 */new GainAbilitySourceEffect(IndestructibleAbility.getInstance(), Duration.EndOfTurn)
+    };
+
+    private static final ContinuousEffect doubles_effect =
+            new GainAbilitySourceEffect(DoubleStrikeAbility.getInstance(), Duration.EndOfTurn);
+
+    Celebr8000Effect() {
+        super(Outcome.Benefit);
+        setText("roll two six-sided dice. For each result of 1, Celebr-8000 gets +1/+1 until end of turn. " +
+                "For each other result, it gains the indicated ability until end of turn. " +
+                "If you rolled doubles, it also gains double strike until end of turn.<br>" +
+                "• 2 — menace<br>" +
```

Comment:
With dot literal:
![image](https://github.com/magefree/mage/assets/17803304/51352c97-3702-44c3-8e29-ddaa925b1207)
With `&bull;`:
![image](https://github.com/magefree/mage/assets/17803304/329947dd-3e77-43aa-b8e4-6f4d0d4f588a)

I can just use the bull character, since that probably plays nicer what with character sets and all.